pom.xml

```diff
@@ -107,6 +107,7 @@
     <module>blueocean-config</module>
     <module>blueocean</module>
     <module>blueocean-jwt</module>
+    <module>blueocean-docker-dev-image</module>
```

Comment:
it's just a matter of copying files to a target directory, so 3 seconds lost maybe